### PR TITLE
Implement standard mySociety footer on writetothem.com

### LINKED ISSUE #307: Implement standard mySociety footer on writetothem.com
Moving across from mysociety/sysadmin#676 which discusses more prominent branding across all of our sites as a way of making their provenance clear.

### PR DESCRIPTION
**Pull request by zarino**
Todo:

- [x] Implement the footer.
- [x] Remove the old `.footer` etc styles.
- [x] Get site description and social links from @MyfanwyNixon.
- [x] Ask @dracos about cobrands (!?)

---

Closes #307.

Because the homepage has a white background, while the rest of the pages have a cream background, we do a very subtle switcheroo of the footer background colour on the homepage vs the other pages, to avoid drawing attention to the footer, or making it look too much like general page content.

screen shot 2016-05-04 at 17 24 11

screen shot 2016-05-04 at 17 24 23

### CODE REVIEW

**Comment by zarino** [comment]
Whoops, just noticed I’ll also need to remove the old `.footer`, `.footer-content`, and `.footer-links-credits li` styles once we’re happy with the new footer. Have added it to the to-do list.

File: web/static/sass/wtt.scss [hunk 1]
```diff
@@ -901,4 +908,33 @@ address {
...
+$mysoc-footer-link-text-color: $link-color;
+$mysoc-footer-link-hover-text-color: #333;
+$mysoc-footer-donate-background-color: $content-background-colour;
+$mysoc-footer-donate-button-background-color: $success-color;
+
+@import "ms_footer";
+
+.mysoc-footer {
+  font-family: $header-font-family;
+  border-top-width: 1px;
```
Comment (zarino) [inline web/static/sass/wtt.scss]: See https://github.com/mysociety/standard-footer/issues/1#issuecomment-215486076 for a potential alternative to setting this width separately.

**Comment by dracos** [comment]
I don't think the last redesign cared about cobrands, so can assume they're not in use any more/ could be removed.

**Comment by struan** [comment]
I think this looks good. Probably should just be one commit though :)

**Pull request merged**